### ISSUE TITLE
Apple Silicon M1 should download corresponding Dart Sdk

### ISSUE THREAD

**Issue by jayjah** [PatOConnor43/asdf-dart#20]
When you're on an Apple Silicon M1, it should download the correct Dart SDK, which is the ARM64 architecture version. 
Currently this cpu architecture is silently ignored and so it downloads the x64 architecture. 

See here:

```
get_arch () {
    local arch=""

    case "$(uname -m)" in
        x86_64|amd64|arm64) arch="x64"; ;;
        i686|i386) arch="ia32"; ;;
        *)
            echo "Arch '$(uname -m)' not supported!" >&2
            ;;
    esac

    echo -n $arch
}
``` 

- ref: [Code](https://github.com/PatOConnor43/asdf-dart/blob/master/bin/install#L30)

**Comment by jayjah** [comment]
This should work for Linux ARMv8 (ARM64) architecture as well, because the download link for the architecture is `arm64` as well. 

One case is still missing: 

- Linux ARMv7: download link correspond to `arm` instead of `arm64`

**Comment by jayjah** [comment]
closed as of merged PR#21
